src/main/kotlin/no/nb/mlt/wls/application/synqapi/synq/SynqBatchMoveItemPayload.kt

```diff
@@ -187,3 +189,18 @@ data class Position(
     )
     val zPosition: Int
 )
+
+fun SynqBatchMoveItemPayload.mapToItemPayloads(): List<MoveItemPayload> {
+    val list = mutableListOf<MoveItemPayload>()
+    for (product in loadUnit) {
+        list.add(
+            MoveItemPayload(
+                hostId = product.productId,
+                hostName = HostName.valueOf(product.hostName.uppercase()),
+                quantity = product.quantityOnHand,
+                location = location
+            )
+        )
+    }
+    return list
```

Comment:
While this would be convenient, you would lose out on the location information, as field comes from the payload and not the product itself.

Reply:
The minimum case for this would look something like this:
```kotlin
fun Product.toPayload(location: String): MoveItemPayload {
    return MoveItemPayload(
        hostId = productId,
        hostName = HostName.valueOf(hostName.uppercase()),
        quantity = quantityOnHand,
        location = location
    )
}

fun SynqBatchMoveItemPayload.mapToItemPayloads(): List<MoveItemPayload> {
    return loadUnit.map { 
        it.toPayload(location)
    }
}
```

Reply:
I think that looks good